RELEASE/scripts/autoscend/paths/wereprofessor.ash

```diff
@@ -0,0 +1,271 @@
+	stomach2	50	stomach1	Thyroxine supplements	Stomach +3
+	stomach3	60	stomach2	Amyloid polypeptide mixture	Stomach +3
+	feed	100	stomach3	Cholecystokinin antagonist	(Unimplemented)
+
+	# Mysticality Skill Tree
+	myst1	10	none	Galanin precipitate	Myst +20%
+	myst2	20	myst1	Cortistatin blocker	Myst +30%
+	myst3	30	myst2	Prolactin inhibitor	Myst +50%
+	bite1	20	myst3	Fluoride rinse	Bite (Stench)
+	bite2	30	bite1	Proton pump eliminator	Add (Sleaze)
+	bite3	40	bite2	Bisphosphonate drip	Increase damage
+	howl	100	bite3	Albuterol innundation	Howl (Stun)
+	res1	20	myst3	Omega-3 megadose	Resist All +20%
+	res2	30	res1	Omega-6 hyperdose	Resist All +20%
+	res3	40	res2	Omega-9 omegadose	Resist All +20%
+	items1	40	res3	Diphenhydramine eyedrops	Item Drop 25%
+	items2	50	items1	Carbinoxamine eye wash	Item Drop +25%
+	items3	60	items2	Intraocular cyproheptadine injections	Item Drop +25%
+	hunt	100	items3	Phantosmic tincture	Hunt (Olfaction)
+	ml1	40	res3	Anabolic megatestosterone	ML +10
+	ml2	50	ml1	Hyperadrenal Pheremones	ML +15
+	ml3	60	ml2	Synthetic Rhabdovirus	ML +25
+	feasting	100	ml3	Peptide catalyst	Regain more HP
+
+	# Moxie Skill Tree
+	mox1	10	none	Dopamine slurry	Mox +20% 
+	mox2	20	mox1	Relaxin balm	Mox +30%
+	mox3	30	mox2	Melatonin suppositories	Mox +50%
+	kick1	20	mox3	Hamstring-tightening solution	Kick (Delevel)
+	kick2	30	kick1	Gluteal 4-Androstenediol inection	Improve Kick
+	kick3	40	kick2	Subcutaneous dimethandrolone implant	Kick (Stun)
+	punt	100	kick3	Novel catecholamine synthesis	Punt (Banish)
+	init1	20	mox3	Adrenal decoction	Init +50%
+	init2	30	init1	Adrenal distillate	Init +50%
+	init3	40	init2	Concentrated adrenaline extract	Init +100%
+	meat1	40	init3	Leptin modulator	Meat Drop +25%
+	meat2	50	meat1	Carnal dehydrogenase infusion	Meat Drop +50%
+	meat3	60	meat2	Dihydrobenzophenanthridine injection	Meat Drop +75%
+	perfecthair	100	meat3	Janus kinase blockers	+5 Stats/Fight
+	liver1	40	init3	Glucagon condensate	Liver +3
+	liver2	50	liver1	Secretin agonist	Liver +3
+	liver3	60	liver2	Synthetic aldosterone	Liver +3
+	pureblood	100	liver3	Synthroid-parathormone cocktail	Shorten ELR
+	*/
+	int[string] rpcost = {"stomach3": 60, "liver3": 60, "stomach2": 50, "liver2":50, "stomach1": 40, "liver1": 40, "hp3": 40, "init3": 40, "hp2": 30, "init2": 30,
+	"hp1": 20, "init1": 20, "mus3": 30, "mox3": 30, "mus2": 20, "mox2": 20, "mus1": 10, "mox1": 10, "punt": 100, "slaughter": 100, "hunt": 100, "kick3": 40, "kick2": 30,
+	"kick1": 20, "rend3": 40, "rend2": 30, "rend1": 20, "items3": 60, "items2": 50, "items1": 40, "res3": 40, "res2": 30, "res1": 20, "myst3": 30, "myst2": 20, "myst1": 10,
+	"bite3": 40, "bite2": 30, "bite1": 20, "perfecthair": 100, "meat3": 60, "meat2": 50, "meat1": 40, "ml3": 60, "ml2": 50, "ml1": 40, "skin3": 60, "skin2": 50, "skin1": 40,
+	"pureblood": 100, "feasting": 100, "skinheal": 100, "howl": 100, "feed": 100};
+	if(do_skills)
+	{
+		auto_log_info("Buying skills", "blue");
+		int cantbuy;
+		while(rp >= 10)
```

Comment:
This was an issue when I initially added skill buying. Will test whether I can remove this while

Reply:
We need the while loop because of the break in the else in the foreach. It could probably be > 0, but >= 10 is also fine because that is the cost of the least expensive skill